src/math/utils.cpp

```diff
@@ -37,14 +37,14 @@ namespace tsid
       vec.tail<9>() = Eigen::Map<const Vector9>(&M.rotation()(0), 9);
     }
 
-    void vectorToSE3(RefVector vec, pinocchio::SE3 & M)
+    void vectorToSE3(RefVector const vec, pinocchio::SE3 & M)
```

Comment:
For passing a constant ref to a vector you can use the `ConstRefVector` type.

Reply:
this file shouldn't have been added to the commit, I'll remove this